module_6/tusk/index.js

```diff
@@ -0,0 +1,40 @@
+"use strict"
+
+// tusk #1
+const randomArr = [1, 2, "Ivan", "banana", "Kiev", "Table", 3, 4];
+
+function arrayCopy () {
+    console.log(randomArr.slice());
```

Comment:
- Щоб функцію можна було перевикористати в майбутньому, потрібно, щоб вона повертала певне значення.
```suggestion
    return array.slice();
```

- Щоб вивести результат в консоль можете нижче (за межами функції) викликати її.
`console.log(arrayCopy(randomArr))`
- Загалом, рішення правильне 👍. Для копіювання простого масиву цілком достатньо методу `.slice()`